src/crawlers/node.js

```diff
@@ -42,7 +42,7 @@ function nodeRecReadDir(roots, {ignore, exts}) {
           }
 
           if (filePath.match(extPattern)) {
-            retFiles.push(filePath);
+            retFiles.push(path.resolve(filePath));
```

Comment:
do we need this? This function is quite hot and is called on thousands of files, so I'd rather avoid it if possible.